Tasks/DownloadBuildArtifact/main.ts

```diff
@@ -0,0 +1,48 @@
+import {BuildArtifact, ArtifactResource} from 'vso-node-api/interfaces/BuildInterfaces';
+import {WebApi, getBearerHandler} from 'vso-node-api/WebApi';
+import * as tl from 'vsts-task-lib/task';
+
+import {ArtifactProvider} from './ArtifactProvider';
+import {FileContainerProvider} from './FileContainer';
+
+async function main(): Promise<void> {
+	let projectId = tl.getVariable('System.TeamProjectId');
+	let buildId = parseInt(tl.getInput("buildId"));
```

Comment:
of course if not supplied no message but if supplied and I give it string is it silently ignored or does user get actionable error and failure?